docs/source/deployment-considerations.rst

```diff
@@ -0,0 +1,91 @@
+
+Most Dask deployment options set reasonable defaults for access (i.e. not making your cluster accessible to anyone on the Internet), but you still should make sure that your clusters (or your users' clusters) aren't accessible to unauthorized users.
+
+Additionally, if you're connecting to a cluster over the Internet, you should ensure that that connection is encrypted, since sensitive information, such as credentials or proprietary data, may flow over it. You might do this by port-forwarding your connection over SSH, using :doc:`TLS <tls>`, or using `Dask-gateway <https://gateway.dask.org/>`_ or a commercial offering that manages this automatically.
+
+
+Controlling cost
+----------------
+It's easy to forget to shut down a cluster and run up an expensive bill over the weekend. Some deployment libraries also may not always be able to fully clean up a cluster—for example, `dask-cloudprovider <https://cloudprovider.dask.org/>`_ won't clean up cloud resources if the client Python process (or machine!) shuts down abruptly.
+
+Therefore, when launching clusters automatically in production, or enabling many team members to launch them, you should be confident that all resources will be cleaned up, or shut down if they exceed a cost threshold.
+
+When managing Dask for a team, you may also want a way to limit how much individual users can spend, to prevent accidental overruns.
+
+
+Monitoring cost
+---------------
+It's good to be able to answer questions such as:
+
+- How much are we spending on Dask?
+- What are we spending it on? (machines, machines that should have been turned off, network egress that shouldn't have happened, etc.)
+- Who/what is responsible?
+
+Non-commercial deployment tools generally don't build in this sort of monitoring. Organizations that need it either end up building their own tools, or turning to commercial deployment offerings.
+
+
+Managing networking
+-------------------
+The Dask client needs to be able to talk to the scheduler, which is potentially on a different system. Users like to be able to access the :doc:`dashboard <dashboard>` from a Web browser. The machines in the cluster need to be able to talk to each other. Typically, whatever :ref:`deployment system <deployment-options>` you use will handle this for you. Sometimes, though, there can be additional considerations around what type of networking to use for best performance. Networking also can have costs associated—cloud providers may charge fixed or usage-based rates for certain types of networking configurations, for example.
+
+You may also have other systems on restricted networks that workers need to access to read and write data, or call APIs. Connecting to those networks could add additional complexity.
+
+Some organizations may have additional network security policies, such as requiring all traffic to be encrypted. Dask supports this with :doc:`TLS <tls>`, which requires additional configuration, and managing certificates.
```

Comment:
Side note but Dask Cloud Provider turns this on by default. I wonder if we should do that in more deployment tooling and make it more of an opt out.